### PR TITLE
add new counter

### PR DESCRIPTION
**Pull request by 0xmchadha**

### CODE REVIEW

**Comment by 0xmchadha** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1604",
    "commit-sha": "95de8beef21cda01d499d4605efd0878527b99f8",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "983",
    "commit-sha": "b9096afeae2f3c1aac3aabc1c5f78f6806840170",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "458",
    "commit-sha": "c17abba71e50c11f1c96f6c9dbe3a11e35f96ad5",
    "pipeline": "master"
  }
]
```

**Comment by 0xmchadha** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "983",
    "commit-sha": "b9096afeae2f3c1aac3aabc1c5f78f6806840170",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "458",
    "commit-sha": "c17abba71e50c11f1c96f6c9dbe3a11e35f96ad5",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1604",
    "commit-sha": "95de8beef21cda01d499d4605efd0878527b99f8",
    "pipeline": "master"
  }
]
```

**Comment by 0xmchadha** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1604",
    "commit-sha": "95de8beef21cda01d499d4605efd0878527b99f8",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "983",
    "commit-sha": "b9096afeae2f3c1aac3aabc1c5f78f6806840170",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "458",
    "commit-sha": "c17abba71e50c11f1c96f6c9dbe3a11e35f96ad5",
    "pipeline": "master"
  }
]
```

**Comment by 0xmchadha** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1604",
    "commit-sha": "95de8beef21cda01d499d4605efd0878527b99f8",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "983",
    "commit-sha": "b9096afeae2f3c1aac3aabc1c5f78f6806840170",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "458",
    "commit-sha": "c17abba71e50c11f1c96f6c9dbe3a11e35f96ad5",
    "pipeline": "master"
  }
]
```

**Comment by 0xmchadha** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1604",
    "commit-sha": "95de8beef21cda01d499d4605efd0878527b99f8",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "983",
    "commit-sha": "b9096afeae2f3c1aac3aabc1c5f78f6806840170",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "458",
    "commit-sha": "09f9d5a283c77636087a874a38d40a6dfbdfe786",
    "pipeline": "master"
  }
]
```

**Comment by 0xmchadha** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1604",
    "commit-sha": "95de8beef21cda01d499d4605efd0878527b99f8",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "983",
    "commit-sha": "b9096afeae2f3c1aac3aabc1c5f78f6806840170",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "458",
    "commit-sha": "09f9d5a283c77636087a874a38d40a6dfbdfe786",
    "pipeline": "master"
  }
]
```

**Comment by 0xmchadha** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1604",
    "commit-sha": "976810f6c8a139dfadca1e4f1af89673f40439a1",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "983",
    "commit-sha": "b9096afeae2f3c1aac3aabc1c5f78f6806840170",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "458",
    "commit-sha": "09f9d5a283c77636087a874a38d40a6dfbdfe786",
    "pipeline": "master"
  }
]
```

**Comment by 0xmchadha** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1604",
    "commit-sha": "976810f6c8a139dfadca1e4f1af89673f40439a1",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "983",
    "commit-sha": "b9096afeae2f3c1aac3aabc1c5f78f6806840170",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "458",
    "commit-sha": "09f9d5a283c77636087a874a38d40a6dfbdfe786",
    "pipeline": "master"
  }
]
```

**Comment by 0xmchadha** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1604",
    "commit-sha": "06305f3c66796de1773aa8c0b4c98da970709223",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "983",
    "commit-sha": "b9096afeae2f3c1aac3aabc1c5f78f6806840170",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "458",
    "commit-sha": "09f9d5a283c77636087a874a38d40a6dfbdfe786",
    "pipeline": "master"
  }
]
```

**Comment by 0xmchadha** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1604",
    "commit-sha": "06305f3c66796de1773aa8c0b4c98da970709223",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "983",
    "commit-sha": "b9096afeae2f3c1aac3aabc1c5f78f6806840170",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "458",
    "commit-sha": "09f9d5a283c77636087a874a38d40a6dfbdfe786",
    "pipeline": "master"
  }
]
```

**Comment by 0xmchadha** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1604",
    "commit-sha": "06305f3c66796de1773aa8c0b4c98da970709223",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "983",
    "commit-sha": "b9096afeae2f3c1aac3aabc1c5f78f6806840170",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "458",
    "commit-sha": "09f9d5a283c77636087a874a38d40a6dfbdfe786",
    "pipeline": "master"
  }
]
```

**Comment by 0xmchadha** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "gaia",
    "pr-id": "458",
    "commit-sha": "09f9d5a283c77636087a874a38d40a6dfbdfe786",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1604",
    "commit-sha": "06305f3c66796de1773aa8c0b4c98da970709223",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "983",
    "commit-sha": "b9096afeae2f3c1aac3aabc1c5f78f6806840170",
    "pipeline": "master"
  }
]
```

**Comment by 0xmchadha** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "gaia",
    "pr-id": "458",
    "commit-sha": "09f9d5a283c77636087a874a38d40a6dfbdfe786",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1604",
    "commit-sha": "06305f3c66796de1773aa8c0b4c98da970709223",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "983",
    "commit-sha": "b9096afeae2f3c1aac3aabc1c5f78f6806840170",
    "pipeline": "master"
  }
]
```

**Comment by 0xmchadha** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "trireme-lib",
    "pr-id": "983",
    "commit-sha": "6c61bac08599db5513e2b328da48c1225e624026",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "458",
    "commit-sha": "09f9d5a283c77636087a874a38d40a6dfbdfe786",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "enforcerd",
    "pr-id": "1604",
    "commit-sha": "06305f3c66796de1773aa8c0b4c98da970709223",
    "pipeline": "master"
  }
]
```

**Comment by 0xmchadha** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "704",
    "commit-sha": "8ae6dd6b2ded0edb5ff6efa951989c9149520df3",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "458",
    "commit-sha": "09f9d5a283c77636087a874a38d40a6dfbdfe786",
    "pipeline": "master"
  }
]
```

**Comment by 0xmchadha** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "704",
    "commit-sha": "8ae6dd6b2ded0edb5ff6efa951989c9149520df3",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "458",
    "commit-sha": "09f9d5a283c77636087a874a38d40a6dfbdfe786",
    "pipeline": "master"
  }
]
```

**Comment by 0xmchadha** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "gaia",
    "pr-id": "458",
    "commit-sha": "09f9d5a283c77636087a874a38d40a6dfbdfe786",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "backend",
    "pr-id": "704",
    "commit-sha": "8ae6dd6b2ded0edb5ff6efa951989c9149520df3",
    "pipeline": "master"
  }
]
```

**Comment by 0xmchadha** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "704",
    "commit-sha": "8ae6dd6b2ded0edb5ff6efa951989c9149520df3",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "458",
    "commit-sha": "f5b490b85a1b18ab7d3be8ab5538ff1aad34c493",
    "pipeline": "master"
  }
]
```

File: specs/counterreport.spec [hunk 1]
```diff
@@ -389,3 +389,9 @@ attributes:
     type: time
     exposed: true
     example_value: "2018-06-14T23:10:46.420397985Z"
+
+  - name: DuplicateAckDrop
```
Comment (t00f) [inline specs/counterreport.spec]: This should be lower cased `duplicateAckDrop`

**Pull request merged**